sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/service/UserService.java

```diff
@@ -27,6 +28,14 @@ public User create(String email, String password) {
 		user.setPassword(password);
 
 		email2User.put(email, user);
+
+		while(true) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add an end condition to this loop. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2189)